app/build.gradle.kts

```diff
@@ -133,9 +133,8 @@ android {
             }
         }
 
-        create("foss") {
-            dimension = "dependencies"
-        }
+        //create("foss") {
+            //?l}
```

Comment:
I know it shows the warning when Gradle syncs but this change is not part of this PR, revert please